codelists/tests/test_api.py

```diff
@@ -171,9 +166,21 @@ def test_versions_post_permission_denied(
     client, user_without_organisation, user_codelist
 ):
     data = {"ecl": "<<128133004"}
-    headers = {"HTTP_AUTHORIZATION": f"Token {user_without_organisation.api_token}"}
 
     with assert_no_difference(user_codelist.versions.count):
-        rsp = client.post(user_codelist.get_versions_api_url(), data, **headers)
+        rsp = post(
+            client,
+            user_codelist.get_versions_api_url(),
+            data,
+            user_without_organisation,
+        )
 
     assert rsp.status_code == 403
+
+
+def post(client, url, data, user):
+    if user is None:
+        headers = {}
+    else:
+        headers = {"HTTP_AUTHORIZATION": f"Token {user.api_token}"}
+    return client.post(url, data, content_type="application/json", **headers)
```

Comment:
Not sure this abstraction gains you much in the long run?  If one is digging into a broken test they now have to look around for this function, when all you're really doing is saving the addition of passing a dictionary to `client.post`!